Sources/Brave/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2718,13 +2735,33 @@ extension BrowserViewController: SessionRestoreScriptHandlerDelegate {
   }
 }
 
+extension BrowserViewController: Web3NameServiceScriptHandlerDelegate {
```

Comment:
Can you move this extension entirely a new file where you can add upcoming web3 related stuff with respect to browser controller?

Reply:
updated in [a4cd1a8](https://github.com/brave/brave-ios/pull/6906/commits/a4cd1a858eb2db471ba0d554b8a31267b8cf01fa)